packages/knowledge/src/base.ts

```diff
@@ -554,7 +554,7 @@ export const base: ChartKnowledgeBase = {
       'A visual representation of a data tree with nodes. Each node is displayed as a rectangle, sized and colored according to values that you assign.',
     purpose: ['Composition', 'Comparison', 'Hierarchy'],
     coord: ['Cartesian2D'],
-    category: ['Statistic'],
+    category: ['Graph'],
```

Comment:
how could we define the boundary between statistic and graph?

Reply:
I also confused with Diagram and Graph.
In https://datavizproject.com/#, diagram includes arc diagram... However, in CKB, arc diagram is Graph.

Reply:
maybe data-cat / info-cat